### PR TITLE
Update `@actions/core` to 1.8.1

### PR DESCRIPTION
**Pull request by smaeda-ks**
following the recent release:

- https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/
- https://github.com/actions/toolkit/blob/main/packages/core/RELEASES.md

### CODE REVIEW

**Comment by smaeda-ks** [comment]
Hi @joshmgross, thoughts on this? Thanks in advance :)

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     }
   },
   "dependencies": {
-    "@actions/core": "^1.6.0",
+    "@actions/core": "^1.8.0",
```
Comment (joshmgross) [inline package.json:34]: Could we update to the latest version `1.8.1`?
https://www.npmjs.com/package/@actions/core/v/1.8.1

Comment (smaeda-ks) [reply package.json]: @joshmgross updated

**Review by joshmgross — APPROVED** [review]

**Comment by joshmgross** [comment]
Thanks!

**Comment by joshmgross** [comment]
This is available in https://github.com/actions/github-script/releases/tag/v6.1.0 and https://github.com/actions/github-script/releases/tag/v6 now

**Comment by smaeda-ks** [comment]
Thank you!

**Pull request merged**